moped-database/migrations/1674748665079_create_layer_council_district/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE layer_council_district (
+    id serial PRIMARY KEY,
+    council_district integer UNIQUE NOT NULL,
+    geography geography ('MULTIPOLYGON', 4326) NOT NULL
+);
```

Comment:
i am very open to ideas about the naming convention here! my thinking was that `layer_council_district` makes clear this is a reference layer, in contrast to, e.g., `feature_signals`.

Reply:
I like this naming - it is easy to spot when scanning through the tables as well. 👍

Reply:
ugh, just realized we left this unfinished conversation. i see both sides. a table is a table that may/may not have a geometry column. i do definitely agree that this is clearly a use case for separate db schema.

in any case, it will still be some time before this table is referenced in application code, so we can definitely change it.